mmv1/third_party/terraform/services/resourcemanager3/resource_resource_manager_capability_test.go.tmpl

```diff
@@ -0,0 +1,60 @@
+package resourcemanager3_test
```

Comment:
This should be inside version guards so that it doesn't appear in the GA provider.

See https://github.com/GoogleCloudPlatform/magic-modules/blob/main/mmv1/third_party/terraform/services/securityscanner/resource_security_scanner_scan_config_test.go.tmpl